docs/protected-sites-import.sh

```
@@ -0,0 +1,466 @@
+# !/bin/bash
```

Shellcheck (0.10.0)

[error] Remove spaces between # and ! in the shebang.

(SC1115)